packages/client/tests/functional/json-null-types/tests.ts

```diff
@@ -63,10 +61,7 @@ testMatrix.setupTestSuite(
                      }
                    })
 
-          Argument json: Provided value Prisma.DbNull of type DbNull on prisma.createOneRequiredJsonField is not a JsonNullValueInput.
-          → Possible values: JsonNullValueInput.JsonNull
-
-
+          Invalid value for argument \`json\`. Expected JsonNullValueInput.
```

Comment:
This might be a bit harder to understand now since `JsonNull` isn't mentioned explicitly. We should look into trying to improve the error message here.

Reply:
(and improving JSON null values DX generally)

Reply:
https://github.com/prisma/team-orm/issues/215